project-file/Muhammad Ibraheem Waheed SP18-BSE-146.md

```diff
@@ -0,0 +1,34 @@
+
+Project Name: Tailor Me
+Project Owner Muhammad Ibraheem
+Full Name
+{Muhammad Ibraheem Waheed}
+
+Roll Number
+{SP18-BSE-146}
+
+Project Details
+Description
+{Tailor Me is a web app plus mobile appp where we intend to connect tailors with customers in real time environment}
+
+Technology Stack
+
+MERN Stack
+MongoDB
+Expres
+Reactjs 
+Nodejs
+OpenCV
+
+Deployment or Distribution Strategy
+{Heroku}
+
+Architecture
```

Comment:
@Ibraheemsheikh2 Please explain the architecture of your project. How different modules are interacting with each other.